app/code/Tasks/FeedbackManager/Controller/Adminhtml/Manage/Decline.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Adminhtml\Manage;
+
+use Magento\Backend\App\Action;
+use Tasks\FeedbackManager\Helper\Sendmail;
+use Magento\Backend\App\Action\Context;
+
+class Decline extends Action
+{
+    protected $feedbackFactory;
+    protected $mailer;
+    
+    public function __construct(
+        Context $context,
+        \Tasks\FeedbackManager\Model\FeedbackFactory $feedbackFactory,
```

Comment:
@gayathri-aravind USe FQDN for classess or don't. Mix and match will impact readability

Reply:
Used FQDN in the class constructors. Commit: 3a2a64414336d13c9a2efa6d09126bb212d438a9